lib/model.js

```diff
@@ -147,19 +151,32 @@ internals.patchSchema = (schema) => {
     return schema.options({ noDefaults: true });
 };
 
+// Converts a Joi error object to the format the Object.ValidationError constructor expects as input
+// https://github.com/Vincit/objection.js/blob/aa3f1a0bb830211e478aa6a664561155c98850f4/lib/model/ValidationError.js#L10
 internals.parseJoiValidationError = (validation) => {
 
     const errors = validation.error.details;
     const validationInfo = {};
 
+    if (errors.length === 1) {
```

Comment:
Why the special case for a single error?  I don't see any comparable code for objection's Ajs validator https://github.com/Vincit/objection.js/blob/aa3f1a0bb830211e478aa6a664561155c98850f4/lib/model/AjvValidator.js#L157-L182

Reply:
ay! yea, my bad there.

there doesn't have to be, I don't think.
I think I did this because of this line: https://github.com/Vincit/objection.js/blob/aa3f1a0bb830211e478aa6a664561155c98850f4/lib/model/ValidationError.js#L16

I separated single and multiple errors because a single error can reliably have an error message without us doing any formatting, so maps directly to the constructor, but for multiple errors, we don't set a message because we can leave that to Objection.Validator's errorsToMessage function.

dunno if that feels sensible at all. just deleting this special case wouldn't change anything, I don't think, except move the single error's type value from the top-level property of the ValidationError to within the data property, like w/ multiple errors.

Should I clean that up?